src/chains/celo/serializers.test.ts

```diff
@@ -26,23 +26,22 @@ const commonBaseTx = {
   feeCurrency: '0x765de816845861e75a25fca122bb6898b8b1282a',
   value: parseEther('1'),
 }
+
 const baseCip42 = {
   ...commonBaseTx,
-  type: 'cip42',
+  gatewayFee: parseGwei('2'),
+  gatewayFeeRecipient: '0x0000000000000000000000000000000000000000',
 } as TransactionSerializableCIP42
 
 const baseCip64 = {
   ...commonBaseTx,
-  type: 'cip64',
+  maxFeePerGas: parseGwei('2'),
+  maxPriorityFeePerGas: parseGwei('2'),
```

Comment:
it looks like commonBase has these fields already.  i guess its fine to be explict